pkg/clock/clock.go

```diff
@@ -0,0 +1,60 @@
+package clock
+
+import (
+	"time"
+
+	bclock "github.com/benbjohnson/clock"
+	"github.com/gavv/monotime"
+)
+
+type (
+	Timer         = bclock.Timer
+	Ticker        = bclock.Ticker
+	MonotonicTime time.Duration
+)
+
+var unixEpoch = time.Unix(0, 0)
+
+type Clock interface {
+	After(d time.Duration) <-chan time.Time
+	AfterFunc(d time.Duration, f func()) *Timer
+	Now() time.Time
+	Since(t time.Time) time.Duration
+	Sleep(d time.Duration)
+	Tick(d time.Duration) <-chan time.Time
+	Ticker(d time.Duration) *Ticker
+	Timer(d time.Duration) *Timer
```

Comment:
What about reusing the interface `clock.Clock` from github.com/benbjohnson/clock for these APIs.